Docs/source/install/hpc/lxplus.rst

```diff
@@ -0,0 +1,136 @@
+.. _building-lxplus:
+
+LXPLUS (CERN)
+=============
+
+The LXPLUS cluster is located at CERN.
+
+* `Lxplus documentation <https://lxplusdoc.web.cern.ch>`__
+* Batch system: `HTCondor <https://batchdocs.web.cern.ch/index.html>`__
+* Filesystem locations:
+    * User folder: ``/afs/cern.ch/user/<a>/<account>`` (10GByte)
+    * Work folder: ``/afs/cern.ch/work/<a>/<account>`` (100GByte)
+    * Eos storage: ``/eos/home-<a>/<account>`` (1T)
+
+Through LXPLUS we have access to CPU and GPU nodes (the latter equipped with NVIDIA A100 or T4 GPUs).
```

Comment:
I find it useful to write how much memory each GPU has, because it makes it easier to translate runs between different machines. Do you like to add this info for the two GPU types?

I usually add this info in a "Running" section, which might still be missing here (example job scripts). See, e.g.,:
- https://warpx.readthedocs.io/en/latest/install/hpc/summit.html#running
- https://warpx.readthedocs.io/en/latest/install/hpc/cori.html#running

Is there anything interesting to add or link for a `Postprocessing` section as well? :)
- https://warpx.readthedocs.io/en/latest/install/hpc/summit.html#post-processing
- https://warpx.readthedocs.io/en/latest/install/hpc/cori.html#post-processing